ERCs/eip-5732/contracts/IERC5732.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+// Author: Zainan Victor Zhou <zzn-ercref@zzn.im>
+// Open source repo: http://zzn.li/ercref
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+
+pragma solidity >=0.7.0 <0.9.0;
+
+/// The EIP-165 identifier of this interface is 0xdd691946
+interface IERC_COMMIT {
```

Comment:
Might as well just call this contract `IERC5732`.

Reply:
We are IERC_COMMIT_CORE and IERC_COMMIT_GENEAL in the EIP and hopefully this matches it. Also I feel it's a bit more readable in the code